From b9850e25586e317c6067fed69691b177cc73f050 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 30 Apr 2018 13:02:24 +0200
Subject: [PATCH 0105/1852] sound: soc: xilinx: Remove empty xilinx_dp_remove()

commit 94e390661d6b8deeafec477dfe529cc83a36abfa from
https://github.com/Xilinx/linux-xlnx.git

There is no reason to have empty function in driver.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xilinx-dp-card.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/sound/soc/xilinx/xilinx-dp-card.c b/sound/soc/xilinx/xilinx-dp-card.c
index 2f1a22623829..7737acda518b 100644
--- a/sound/soc/xilinx/xilinx-dp-card.c
+++ b/sound/soc/xilinx/xilinx-dp-card.c
@@ -96,11 +96,6 @@ static int xilinx_dp_probe(struct platform_device *pdev)
 	return 0;
 }
 
-static int xilinx_dp_remove(struct platform_device *pdev)
-{
-	return 0;
-}
-
 static const struct of_device_id xilinx_dp_of_match[] = {
 	{ .compatible = "xlnx,dp-snd-card", },
 	{},
@@ -114,7 +109,6 @@ static struct platform_driver xilinx_dp_aud_driver = {
 		.pm		= &snd_soc_pm_ops,
 	},
 	.probe	= xilinx_dp_probe,
-	.remove	= xilinx_dp_remove,
 };
 module_platform_driver(xilinx_dp_aud_driver);
 
-- 
2.31.1

